TFLite_detection_stream.py

```diff
@@ -325,7 +383,12 @@ def detection():
         boxes_idx, classes_idx, scores_idx = 0, 1, 2
 
     # Initialize video stream
-    videostream = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL).start()
+    DRIVEWAY = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL).start()
+    FRONT_PORCH = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL.replace("channel=1", "channel=2")).start()
+    SW_YARD = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL.replace("channel=1", "channel=3")).start()
+    W_PORCH = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL.replace("channel=1", "channel=4")).start()
+    N_YARD = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL.replace("channel=1", "channel=5")).start()
+    NE_YARD = VideoStream(resolution=(imW,imH), framerate=30, stream_url=STREAM_URL.replace("channel=1", "channel=6")).start()
```

Comment:
We should utilize stream URL from vault to reduce issues with invalid streams or sensitive data loss.